test/integration/tests/button/multiple.js

```diff
@@ -30,32 +30,32 @@ for (const flow of ["popup", "iframe"]) {
     });
 
     const cases = [
-      {
```

Comment:
Why were these commented out?

Reply:
They shouldn't be. We ran the full test suite locally and this test was failing and we accidentally committed part of our debug process. TLDR it's test flake locally. They all pass in CI.